fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/debug/Debugger.java

```diff
@@ -215,7 +215,9 @@ enum Location {
         MATCH_PRE,
         YIELD,
         FAILURE,
-        COUNT
+        COUNT,
+        NEW,
+        REUSED
```

Comment:
`REUSED` is a metric to show how many `yield()` were actually not inserted but reused.